Short-Answer/Algorithms_Questions.md

```diff
@@ -19,6 +20,9 @@ b)  sum = 0
       while j < n:
         j *= 2
         sum += 1
+
+  #Runtime is 0(n) because it is increases linar to what the input is
```

Comment:
We multiply the individual runtimes of each loop to obtain the runtime of an entire block since these loops are nested, giving us an overall runtime of O(n log(n)).